private/bufpkg/bufcheck/buflint/testdata/protovalidate_rules/oneof.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package what;
+
+import "buf/validate/validate.proto";
+import "google/protobuf/duration.proto";
+
+message Ds {
+  oneof haha {
```

Comment:
lol

Reply:
fixed in [cdc4b77](https://github.com/bufbuild/buf/pull/2528/commits/cdc4b774ede852c5476bd9c66089c6e8163ffc7f) 
lol, forgot to update its name after copying it from scratch space